object-store-client-common/src/main/scala/uk/gov/hmrc/objectstore/client/model.scala

```diff
@@ -23,17 +23,13 @@ sealed trait Path { def asUri: String }
 
 object Path {
   case class Directory(value: String) extends Path {
-    override def asUri: String =
-      if (value.endsWith("/")) value else value + "/"
-
-    def file(fileName: String): File =
```

Comment:
Removed as unused.

Reply:
file is a public function, used by clients - shouldn't remove

Reply:
fair enough, I can bring that back. I deleted it as we ourselves never used to create test object.